nixos/modules/misc/documentation.nix

```diff
@@ -255,6 +255,23 @@ in
         '';
       };
 
+      nixos.options.allowDocBook = mkOption {
+        type = types.bool;
+        default = true;
+        description = lib.mdDoc ''
+          Whether to allow DocBook option docs. When set to `false` all option using
+          DocBook documentation will cause a manual build error; additionally a new
+          renderer may be used.
+
+          ::: {.note}
+          The `false` setting for this option is not yet fully supported. While it
+          should work fine and produce the same output as the previous toolchain
+          using DocBook it may not work in all circumstances. It is always permissible
+          to set this to `true` when it was set to `false` previously.
```

Comment:
Wait what does this mean? "It is always permissible to set this to `true` when it was set to `false` previously."?

Reply:
I see, how about rewording this a bit then, something like "Whether markdown option documentation is allowed is independent of this option."